media/vp8_ivf.ksy

```diff
@@ -49,23 +50,62 @@ seq:
 
 ## payload start
   - id: image_data
-    type: blocks
+    type: frames
     repeat: expr
     repeat-expr: num_frames
 ## payload end
 
 ## type definitions
 types:
-  blocks:
+  frames:
     seq:
       - id: entries
-        type: block
-  block:
+        type: frame
+  frame:
     seq:
       - id: len_frame
         doc: size of the frame data
         type: u4
       - id: timestamp
         type: u8
-      - id: framedata
-        size: len_frame
+      - id: is_interframe
+        doc: A 1-bit frame type (0 for key frames, 1 for interframes).
+        type: b1 
+      - id: version_number
+        doc: >
+          A 3-bit version number (0 - 3 are defined as four different 
+          profiles with different decoding complexity; other values may be 
+          defined for future variants of the VP8 data format).
+        type: b3
+      - id: flag_show_frame
+        doc: >
+          A 1-bit show_frame flag (0 when current frame is not for display,
+          1 when current frame is for display).
+        type: b1
+      - id: first_data_partition_size
+        doc: >
+          A 19-bit field containing the size of the first data partition in
+          bytes.
+        type: b19
+      - id: keyframe_hdr
+        type: keyframe_tag
+        if: not is_interframe
+      - id: compressed_keyframe
+        size: len_frame - 10
+        if: not is_interframe
+      - id: compressed_interframe
+        size: len_frame - 3
+        if: is_interframe
+  keyframe_tag:
+    seq:
+      - id: start_code
+        type: b24
+      - id: width
+        type: b14
+      - id: horizontal_scale
+        type: b2
+      - id: height
+        type: b14
+      - id: vertical_scale
+        type: b2
+        
```

Comment:
There should not be any whitespace "after the newline at EOF" (actually not even at the end of _any_ line - that's why we have the [`trim_trailing_whitespace`](https://github.com/kaitai-io/kaitai_struct_formats/blob/9aa3c997fe6555c3c4e79ec34cd9c108960d1d26/.editorconfig#L8) option in [`.editorconfig`](https://editorconfig.org/) for .ksy files, but not many people have their editors configured to honor it):

```suggestion

```

Reply:
* I used the rfc6386 as well. It is the "official" specification I think. (regarding source)
* I see. I will configure my editor accordingly. (regarding trailing whitespace)